codegen/src/test/resources/software/amazon/awssdk/codegen/poet/model/sharedstream/eventstream.java

```diff
@@ -41,6 +41,13 @@ public void accept(StreamDeathsResponseHandler.Visitor visitor) {
         return DefaultPerson.builder();
     }
 
+    /**
+     * The type of this event. Corresponds to the {@code :event-type} header on the Message.
+     */
+    default String sdkEventType() {
```

Comment:
Should we use enum for the type?

Reply:
What's the benefit?

Reply:
It'll be easier to enumerate all types when they do `switch`, whereas with string, they will need to manually check the code to see all available types.

Reply:
Sounds good.